org.kde.CrowTranslate.yaml

```diff
@@ -0,0 +1,87 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+add-extensions:
+  org.kde.CrowTranslate.tessdata:
+    directory: share/tessdata
+    version: stable
+    versions: stable;beta;test
+    no-autodownload: true
+    autodelete: true
+finish-args:
+  - --device=dri
+  - --env=TESSDATA_PREFIX=/app/share
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+copy-icon: true
+command: crow
+separate-locales: false
```

Comment:
Why is this disabled?

Reply:
The is unable to load localization without this line.

Reply:
Are you sure? You need to install the `.Locale` extension for that locale. By default Flatpak will only install the locale extension for your host locale.

This disabled creating locale extension and bundles all locales into the app.

Reply:
Ah, I didn't know about it! In my application I have locale selection and it could be confusing when user picks a localization and it doesn't work.
Should I keep `separate-locales: false` or remove it?

Reply:
Then keep it, usually apps follow the system locale.